.travis.yml

```diff
@@ -204,31 +181,33 @@ before_install:
       fi
 
      # Check CC variable
-    - echo $CC
+    - echo "CC="$CC
 
     # Write configuration items to standard location to make sure they are
     # ignored (the tests will fail if not)
     - mkdir -p $HOME/.astropy/config/
     - printf "unicode_output = True\nmax_width = 500" > $HOME/.astropy/config/astropy.cfg
 
+    - if [[ $TOXENV == *-cov ]]; then
+        export TOXPOSARGS=$TOXPOSARGS" --cov-report=xml:"$TRAVIS_BUILD_DIR"/coverage.xml";
+      fi
+
+    - echo "TOXENV="$TOXENV
+    - echo "TOXARGS="$TOXARGS
+    - echo "TOXPOSARGS="$TOXPOSARGS
 
 install:
-    - if [[ $USE_CI_HELPERS == True ]]; then
+    - if [[ $TRAVIS_OS_NAME == osx || $TRAVIS_OS_NAME == windows ]]; then
```

Comment:
Travis provides a usable python on linux but not windows or mac

Reply:
It's the opposite - it's not needed on Linux because then you can just set up Python with ``language: python``, but that isn't an option for Windows or MacOS X so we need to use conda there.